test/Microsoft.AspNetCore.Routing.Performance/RoutingBenchmark.cs

```diff
@@ -35,7 +35,7 @@ public RoutingBenchmark()
                 NullLoggerFactory.Instance,
                 UrlEncoder.Default,
                 new DefaultObjectPool<UriBuildingContext>(new UriBuilderContextPooledObjectPolicy(UrlEncoder.Default)),
-                new DefaultInlineConstraintResolver(new OptionsManager<RouteOptions>(Enumerable.Empty<IConfigureOptions<RouteOptions>>())));
+                new DefaultInlineConstraintResolver(new OptionsManager<RouteOptions>(new OptionsFactory<RouteOptions>(Enumerable.Empty<IConfigureOptions<RouteOptions>>(), Enumerable.Empty<IPostConfigureOptions<RouteOptions>>()))));
```

Comment:
Would it be possible to add a default constructor to this? We have a million places in MVC where we've done stuff like this